.vib/git/goss/git.yaml

```diff
@@ -0,0 +1,7 @@
+command:
```

Comment:
These tests don't seem to be retriable.

Reply:
I've added a random folder

Reply:
Have you tested this? IIRC, using `$RANDOM` was not possible in Goss (note it is a bash function, not and env). Please review this; the alternative we have always used here is the go template function `randAlpha`.